lib/extractor/index.ts

```diff
@@ -203,27 +203,25 @@ function layersWithLatestFileModifications(
     // go over extracted files products found in this layer
     for (const filename of Object.keys(layer)) {
       // if finding a deleted file - trimming to its original file name for excluding it from extractedLayers
-      if (isDeletedFile(filename)) {
+      if (isWhitedOutFile(filename)) {
```

Comment:
If the current file is a whited-out file, wouldn't that mean we can also `continue` as the next statements won't apply for sure?